bzt/engine.py

```diff
@@ -1349,15 +1363,16 @@ def get_data_sources(self):
 
             yield source
 
-    def get_requests(self, parser=RequestParser, require_url=True):
+    def get_requests(self, parser=RequestParser, require_url=True, transactions=False):
```

Comment:
Can we be more specific about naming this flag? `transactions` is kind of vague, it really should be something like `flatten_transactions` or `remove_transactions`.